### PR TITLE
feat: Move duration for poll into config file

### LINKED ISSUE #63: Move polling Duration into config file

### PR DESCRIPTION
**Pull request by d3v3us**
Closes #63

### CODE REVIEW

File: croncat/src/config/mod.rs [hunk 1]
```diff
@@ -18,6 +18,7 @@ pub struct ChainConfig {
     pub contract_address: String,
     pub gas_prices: f64,
     pub gas_adjustment: f64,
+    pub polling_duration_secs:u64,
```
Comment (SeedyROM) [inline croncat/src/config/mod.rs]: Could you follow the instructions in the README on how to setup the git hooks so we don't push unformatted code.

File: croncat/src/system/mod.rs [hunk 2]
```diff
@@ -30,6 +30,7 @@ pub async fn run(
     signer: GrpcSigner,
     initial_status: AgentStatus,
     with_rules: bool,
+    polling_duration_secs:u64,
```
Comment (SeedyROM) [inline croncat/src/system/mod.rs]: Same here.

File: croncatd/src/main.rs [hunk 3]
```diff
@@ -157,7 +159,12 @@ async fn main() -> Result<(), Report> {
             // Create a channel to handle graceful shutdown and wrap receiver for cloning
             let (shutdown_tx, shutdown_rx) = channels::create_shutdown_channel();
             // Start the agent
-            system::run(shutdown_tx, shutdown_rx, signer, initial_status, with_rules).await?;
+            system::run(shutdown_tx,
```
Comment (SeedyROM) [inline croncatd/src/main.rs]: Yeah same here too, the rest of the function params that is. Can't edit the range now.

**Review by SeedyROM — CHANGES_REQUESTED** [review]
See note about setting up the git hooks so you aren't commiting unformatted code.

File: croncat/src/system/mod.rs (hunk 2 above)
Comment (d3v3us) [reply croncat/src/system/mod.rs]: thank you)

**Review by SeedyROM — APPROVED** [review]
LGTM! :hammer_and_pick:

**Review by TrevorJTClarke — APPROVED** [review]
Very much needed, thank you!

**Pull request merged**